src/Modules/Login.php

```diff
@@ -143,6 +144,15 @@ public function authenticate( $user = null ) {
 			if ( $user instanceof WP_User ) {
 				$this->authenticated = true;
 
+				/**
+				 * Fires once the user has been authenticated via Google OAuth.
+				 *
+				 * @param WP_User $user WP User object.
+				 *
+				 * @since 1.2.3
```

Comment:
Ideally `@since` should be above `@param`. Also add blank line after `@since`. I missed this in last review. Apologies.